include/boost/geometry/formulas/karney_inverse.hpp

```diff
@@ -448,9 +449,9 @@ class karney_inverse
 
         if (swap_point < 0)
         {
-            swap(sin_alpha1, sin_alpha2);
-            swap(cos_alpha1, cos_alpha2);
-            swap(result.geodesic_scale, M21);
+            boost::core::invoke_swap(sin_alpha1, sin_alpha2);
+            boost::core::invoke_swap(cos_alpha1, cos_alpha2);
+            boost::core::invoke_swap(result.geodesic_scale, M21);
```

Comment:
We can, if I see it correctly, replace these calls (here) with just `std::swap`
And maybe on more places.
But for now this PR looks okay to me.